projects/mbedtls/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build project
+cmake .
+make -j$(nproc) all
+
+# build corpuses
+cd tests
+zip -r fuzz/fuzz_x509crl_seed_corpus.zip data_files/crl*
+zip -r fuzz/fuzz_x509crt_seed_corpus.zip data_files/*.crt data_files/dir*/*.crt fuzz/corpuses/x509/*
+zip -r fuzz/fuzz_x509csr_seed_corpus.zip data_files/*.csr data_files/*.req.*
+zip -r fuzz/fuzz_privkey_seed_corpus.zip data_files/*.key data_files/*.pem
+zip -r fuzz/fuzz_pubkey_seed_corpus.zip data_files/*.pub data_files/*.pubkey data_files/*pub.pem
+
+cd fuzz
+# export other associated stuff
+cp *.options $OUT/
+cp fuzz_*_seed_corpus.zip $OUT/
+
+# build fuzzers
```

Comment:
this is fine, but why not a loop?
all these seem to be the same (modulo the fuzz target name)

Reply:
Indeed, I am changing that